core/src/main/scala/org/apache/spark/scheduler/ShuffleMapStage.scala

```diff
@@ -87,7 +87,10 @@ private[spark] class ShuffleMapStage(
    */
   def isAvailable: Boolean = numAvailableOutputs == numPartitions
 
-  /** Returns the sequence of partition ids that are missing (i.e. needs to be computed). */
+  /**
+   * Returns the sequence of partition ids that are missing (i.e. needs to be computed).
+   * If the current stage is indeterminate, missing partition is all partitions every time.
```

Comment:
this is true in the places where this method happens to be called, but its a little misleading -- if you called it in the middle of a normal run of a stage, it would tell you what was left.  I think its better to not put this comment in as it could be wrong in the future.

Reply:
Copy that, thanks for the advice, move this comment in https://github.com/apache/spark/pull/24892/files/eb5c58b35427497accb43c47423c23c48810aa78#diff-6a9ff7fb74fd490a50462d45db2d5e11R1103, I think that's a more appropriate place.